zh/backup-to-s3.md

```diff
@@ -16,35 +16,82 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
 
 目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文对 Ceph 和 Amazon S3 这两种存储的使用进行描述。本文档提供如下备份示例。示例假设对部署在 Kubernetes `test1` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，下面是具体操作过程。
 
+### AWS 账号的三种权限授予方式
+
+如果使用 Amazon S3 来备份恢复集群，可以使用三种权限授予方式授予权限，参考[使用 BR 工具备份 AWS 上的 TiDB 集群](backup-to-aws-s3.md#aws-账号权限授予的三种方式)，使用 Ceph 作为后端存储测试备份恢复时，是通过 AccessKey 和 SecretKey 模式授权。
+
 ### Ad-hoc 全量备份环境准备
 
-1. 下载文件 [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `test1` 这个 namespace 中创建备份需要的 RBAC 相关资源：
+参考[Ad-hoc 全量备份环境准备](backup-to-aws-s3.md#ad-hoc-全量备份环境准备)
+
+### 备份数据到兼容 S3 的存储
+
++ 创建 `Backup` CR，通过 accessKey 和 secretKey 授权的方式将数据备份到 Amazon S3：
```

Comment:
Please keep the same format for all of the `accessKey` `secretKey`.

Reply:
done